internal/formatter.go

```diff
@@ -136,23 +132,23 @@ func SpanLine(name string, startMillis, durationMillis int64, source, traceId, s
 		source = defaultSource
 	}
 
-	if !isUUIDFormat(traceId) {
-		return "", errors.New("traceId is not in UUID format")
+	if !isUUIDFormat(traceID) {
+		return "", errors.New("traceID is not in UUID format")
 	}
-	if !isUUIDFormat(spanId) {
-		return "", errors.New("spanId is not in UUID format")
+	if !isUUIDFormat(spanID) {
+		return "", errors.New("spanID is not in UUID format")
 	}
 
 	sb := GetBuffer()
 	defer PutBuffer(sb)
 
 	sb.WriteString(sanitizeValue(name))
 	sb.WriteString(" source=")
-	sb.WriteString(sanitizeValue(source))
-	sb.WriteString(" traceId=")
-	sb.WriteString(traceId)
-	sb.WriteString(" spanId=")
-	sb.WriteString(spanId)
+	sb.WriteString(strconv.Quote(sanitizeInternal(source)))
+	sb.WriteString(" traceID=")
```

Comment:
The line format at the proxy looks for `traceId` and `spanId`. Let's revert these changes.